OpenRA.Mods.Common/Traits/Buildings/PlaceBuildingVariants.cs

```diff
@@ -21,6 +21,9 @@ public class PlaceBuildingVariantsInfo : TraitInfo<PlaceBuildingVariants>, Requi
 		[Desc("Variant actors that can be cycled between when placing a structure.")]
 		public readonly string[] Actors = null;
 
+		[Desc("Facings reference of variant actors for BaseBuilderBotModule. Should always have the length of number of Variant Actors + 1, due to the first element refer to actor itself")]
```

Comment:
I think we can remove ` for BaseBuilderBotModule` since this could be used for other purposes if we wanted.

Reply:
fixed